File: euclidean.c
#include <stdio.h>

int getGCD(int m, int n) {
  while (n != 0) {
    int r = m % n;
    m = n;
    n = r;
  }
  return m;
}

int main() {
  printf("%d\n", getGCD(40, 232));
  return 0;
}

File: euclidean.py
def getGCD(m, n):
  while n != 0:
    r = m % n
    m = n
    n = r
  return m

def gcd(m, n):
  while n:
    m, n = n, m % n
  return m

print(getGCD(40, 232))
print(gcd(40, 232))
